desktop/storeService.ts

```diff
@@ -1,11 +1,26 @@
-const Store = require('electron-store');
+import { nativeTheme } from 'electron';
+import Store from 'electron-store';
+import { TypedSchemaStore } from '../shared/types';
 
 class StoreService {
-  static store: typeof Store;
+  public static store: Store<TypedSchemaStore>;
 
   static init() {
     if (!StoreService.store) {
-      StoreService.store = new Store();
+      StoreService.store = new Store<TypedSchemaStore>({
```

Comment:
this schema will be hard to support cause there are a lot of changes ahead what makes it redundant at this point

Reply:
I'm sure that this is the best way to avoid shooting in the leg and move fast. If something will change — it will require a change in the interface and other places without producing bugs. So, I believe that "there are a lot of changes ahead" is the argument for providing types :)